test/e2e/e2e_go_suite_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_test
+
+import (
+	"fmt"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	kbtestutils "sigs.k8s.io/kubebuilder/test/e2e/utils"
+
+	. "github.com/onsi/ginkgo" //nolint:golint
+	. "github.com/onsi/gomega" //nolint:golint
+
+	testutils "github.com/operator-framework/operator-sdk/test/internal"
+)
+
+// TestE2EGo ensures the Go projects built with the SDK tool by using its binary.
+func TestE2EGo(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping Operator SDK E2E Go Suite testing in short mode")
+	}
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "E2EGo Suite")
+}
+
+var (
+	tc testutils.TestContext
+	// isPromethuesManagedBySuite is true when the suite tests is installing/uninstalling the Prometheus
```

Comment:
just a type-o catch, all instances of 'Promethues' should be changed to 'Prometheus'

Reply:
fixed.